app/src/gplay/AndroidManifest.xml

```diff
@@ -11,11 +11,12 @@
         android:networkSecurityConfig="@xml/network_security_config"
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
-        tools:replace="label, icon, theme, name, allowBackup">
+        tools:replace="label, icon, theme, name, allowBackup"
+        tools:ignore="UnusedAttribute, ExportedService">
```

Comment:
Wouldn't it be better to remove the unused attributes, instead of ignoring them for lint?

Reply:
It's about networkSecurityConfig which is used only on 24 and up :)

I don't want to make a separate Manifest for 24 and up.